packages/editor-ui/src/mixins/workflowRun.ts

```diff
@@ -240,8 +264,6 @@ export const workflowRun = defineComponent({
 					await this.workflowHelpers.saveCurrentWorkflow();
```

Comment:
We have to move this block up as well, because if not the click to "Execute workflow" of new (unsaved) workflows is now broken

Reply:
But how far? Should it be the first line of the function?
Now you mention this I don't even know why it was in the middle?

Reply:
You can put it right above the `const workflowData = await this.workflowHelpers.getWorkflowDataToSave();` line, just like it was in here

Reply:
Ok, done